packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -391,7 +391,8 @@ async function handler(req: CustomRequest) {
       data: {
         status: BookingStatus.CANCELLED,
         cancellationReason: cancellationReason,
-        iCalSequence: evt.iCalSequence,
+        // Assume that canceling the booking is the last action
+        iCalSequence: evt.iCalSequence || 100,
```

Comment:
`nitpick`: Nullish coalescing, could 0 be explicitly set?

Reply:
When calendars parse `.ics` files they look at the sequence for the latest update. This sequence starts at 0. In the case of cancelling bookings, it should be the last update for the event, so I wanted to fall back to a sequence number that would most likely be the highest.

Reply:
THis comment is important enough to go inline in the code.